### PR TITLE
I changed the initial liters of product to 0.1 because user can't ord…

### PR DESCRIPTION
**Pull request by eilonasher**
…er 0 liters, and can't press "add to my cart" when there are 0 liters so I've made a toast if he did it and canceled the product from being added. make your notes

### CODE REVIEW

File: app/src/main/java/com/example/sabonit/ProductActivity.java [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import androidx.core.content.res.ResourcesCompat;
 
 import android.annotation.SuppressLint;
+import android.content.Context;
```
Comment (yairelboher) [inline app/src/main/java/com/example/sabonit/ProductActivity.java:10]: changes to make a toast, great

File: app/src/main/java/com/example/sabonit/ProductActivity.java [hunk 2]
```diff
@@ -69,7 +71,7 @@ public void onCheckedChanged(RadioGroup group, int checkedId) {
         Bundle bundle = getIntent().getExtras();
         db = FirebaseFirestore.getInstance();
         // default value when initializing the seek bar
-        initialProgress = 0;
+        initialProgress = 1;
```
Comment (yairelboher) [inline app/src/main/java/com/example/sabonit/ProductActivity.java:74]: it's smart, exaactly what we talked about

File: app/src/main/java/com/example/sabonit/ProductActivity.java [hunk 3]
```diff
@@ -231,6 +233,16 @@ public void goToCart(View view) {
...
+        if(currentProductLiters == 0)
+        {
+            Context context = getApplicationContext();
+            CharSequence text = "You can't add 0 liters!";
+            int duration = Toast.LENGTH_LONG;
+
+            Toast toast = Toast.makeText(context, text, duration);
+            toast.show();
+            return;
+        }
```
Comment (yairelboher) [inline app/src/main/java/com/example/sabonit/ProductActivity.java:245]: yes, it was strange if someone can add o liters

**Review by yairelboher — COMMENTED** [review]
from me its you to merge it, good job

File: app/src/main/java/com/example/sabonit/ProductActivity.java (hunk 2 above)
Comment (galpinto25) [reply app/src/main/java/com/example/sabonit/ProductActivity.java:74]: OK nice, we can check if there is min value for seek bar

File: app/src/main/java/com/example/sabonit/ProductActivity.java (hunk 3 above)
Comment (galpinto25) [reply app/src/main/java/com/example/sabonit/ProductActivity.java:245]: yes you are right

**Pull request merged**